packages/components/src/Gesture/withListGesture.js

```diff
@@ -71,9 +85,16 @@ function onKeyDown(event, ref) {
 
 export default function withListGesture(WrappedComponent) {
 	function ListGesture(props) {
-		return <WrappedComponent {...props} onKeyDown={onKeyDown} />;
+		const { loop } = props;
```

Comment:
This must not be in a prop, as it must not change in time. A component would want he loop or not.
Take it in `withListGesture` arguments `withListGesture(WrappedComponent, loop)`

Reply:
fixed as suggested.